spec/controllers/concerns/administrate/punditize_spec.rb

```diff
@@ -2,15 +2,32 @@
 
 module Administrate
   RSpec.describe Punditize do
-    it "exists when 'Pundit' is not defined" do
-      unload_constants do
+    if Pundit.const_defined?(:Authorization)
```

Comment:
Not sure how we feel about conditionals in spec files? Are there better alternatives?